### DISCUSSION TITLE
Error when attempting to install modules on Bookwork

### DISCUSSION THREAD

**Discussion by WanglerDangler** [AllskyTeam/allsky-modules#31] [Q&A] [answered]
Hello, I installed allsky on Pi OS Bookworm, using --branch Point-Release-4

When attempting to install the modules though (need the dewheater!), I keep getting "externally-managed-environment" errors, and the install won't continue.

I am new to Pi and Linux overall so I am not sure how to fix. 

Is this possibly an issue with Bookworm vs an older version of the Pi OS?

Thank you in advance for your help!

**Comment by Alex-developer** [comment, marked as answer]
Use the dev branch for the modules. This has the changes for Bookwork

**Reply by WanglerDangler** [reply]
Excellent!  Thank you!  I had seen another solution, but it was a bandaid.

This is much better!

**Reply by Alex-developer** [reply]
No problem :-)
